readme.md

```diff
@@ -94,7 +108,7 @@ How is the concept of OAuth related to a valet key?
 
 Your Answer:
 ```text
-
+You want to use their valet service so you give them your key (app ID and app secret) and in exchange they take your car and give you something to identify which car is yours (access token)
```

Comment:
:+1:
Nice.  Also, the valet key gives them limited access to your car (they can't open trunk/glovebox).